CONTRIBUTING.md

```diff
@@ -90,6 +90,34 @@ Appwrite uses an internal micro-framework called Litespeed.js to build simple UI
 
 After finishing the installation process, you can start writing and editing code.
 
+
+#### Advanced Topics
+We love to create issues that are good for begginers and label them as `good for begginers` or `hacktoberfest`, but some more advanced topics might require extra knowledge. Below is a list with links for which you can use to learn more about some of them more advance topics that will help you master the Appwrite codebase.
```

Comment:
*beginners not begginers

* Below is a list **with links which** you can use to learn more about some **of the more** advance topics that will help you master Appwrite codebase.